pkg/connector/matrix.go

```diff
@@ -65,23 +67,46 @@ func (l *LinkedInClient) HandleMatrixMessage(ctx context.Context, msg *bridgev2.
 
 	if msg.Content.MsgType.IsMedia() {
 		err := l.main.Bridge.Bot.DownloadMediaToFile(ctx, msg.Content.URL, msg.Content.File, false, func(f *os.File) error {
-			attachmentType := linkedingo.MediaUploadTypePhotoAttachment
-			if msg.Content.MsgType != event.MsgImage {
-				attachmentType = linkedingo.MediaUploadTypeFileAttachment
-			}
 
-			if msg.Content.MsgType == event.MsgVideo {
+			attachmentType := linkedingo.MediaUploadTypeFileAttachment
+			switch msg.Content.MsgType {
+			case event.MsgImage:
+				attachmentType = linkedingo.MediaUploadTypePhotoAttachment
+			case event.MsgVideo:
 				attachmentType = linkedingo.MediaUploadTypeVideoAttachment
+			case event.MsgAudio:
+				attachmentType = linkedingo.MediaUploadTypeVoiceMessage
 			}
 
 			filename := getMediaFilename(msg.Content)
+			if msg.Content.MsgType == event.MsgAudio {
+				if !ffmpeg.Supported() {
+					return errors.New("ffmpeg is required to send voice message")
```

Comment:
ffmpeg is only required for converting. For voice messages, audio/ogg should be converted and the correct type (I guess audio/mp4 here) should pass through. Also, only messages with the MSC3245Voice flag should be flagged as voice messages and converted, other `m.audio` messages should be sent as files.

Finally, https://github.com/mautrix/linkedin/blob/main/pkg/connector/capabilities.go#L116-L118 looks wrong and should be updated to match reality, check the other bridges for how they conditionally allow audio/ogg based on ffmpeg availability.